### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by statasaurus**

### CODE REVIEW

**Comment by mstackhouse** [comment]
- [ ] Ignore the .DS_store file and remove it from git

I'm also getting an error from the Building Specification Readers vignette:

```r
spec_to_metacore(metacore_example("mock_spec.xlsx"))
Error: Unable to rename the following columns in Domains:                                                                            
 label matches 2 columns
Please check your regular expression 
Called from: str_c(names(nm_test), " matches ", nm_test, " columns") %>% str_c(collapse = "\n ") %>% 
    paste0("Unable to rename the following columns in ", names(matches[1]), 
        ":\n ", ., "\nPlease check your regular expression ") %>% 
    stop(call. = FALSE)
```

This comes from here: https://github.com/atorus-research/metacore/blob/524ab84b03451ded71e74395252693361558faaf/R/spec_builder.R#L689

**Comment by statasaurus** [comment]
> * [ ]  Ignore the .DS_store file and remove it from git
> 
> I'm also getting an error from the Building Specification Readers vignette:
> 
> ```r
> spec_to_metacore(metacore_example("mock_spec.xlsx"))
> Error: Unable to rename the following columns in Domains:                                                                            
>  label matches 2 columns
> Please check your regular expression 
> Called from: str_c(names(nm_test), " matches ", nm_test, " columns") %>% str_c(collapse = "\n ") %>% 
>     paste0("Unable to rename the following columns in ", names(matches[1]), 
>         ":\n ", ., "\nPlease check your regular expression ") %>% 
>     stop(call. = FALSE)
> ```
> 
> This comes from here:
> 
> https://github.com/atorus-research/metacore/blob/524ab84b03451ded71e74395252693361558faaf/R/spec_builder.R#L689

What versions of stuff where you using?

File: dev/convert_to_xml2.R [hunk 1]
```diff
@@ -0,0 +1,241 @@
+library(xml2)
```
[the remark this answers is not in the document]
Comment (statasaurus) [reply dev/convert_to_xml2.R:1]: Yeah I was going between computers so I just needed it. But I can clean up the repo

[the remark this answers is not in the document]
Comment (statasaurus) [reply dev/convert_to_xml2.R:1]: I am going to delete the dev folder

**Comment by mstackhouse** [comment]
> 

@statasaurus look at me being a bad reviewer...

```
R version 4.0.2 (2020-06-22)
Platform: x86_64-pc-linux-gnu (64-bit)
Running under: Ubuntu 18.04.6 LTS

Matrix products: default
BLAS:   /usr/lib/x86_64-linux-gnu/openblas/libblas.so.3
LAPACK: /usr/lib/x86_64-linux-gnu/libopenblasp-r0.2.20.so

locale:
 [1] LC_CTYPE=C.UTF-8       LC_NUMERIC=C           LC_TIME=C.UTF-8        LC_COLLATE=C.UTF-8     LC_MONETARY=C.UTF-8   
 [6] LC_MESSAGES=C.UTF-8    LC_PAPER=C.UTF-8       LC_NAME=C              LC_ADDRESS=C           LC_TELEPHONE=C        
[11] LC_MEASUREMENT=C.UTF-8 LC_IDENTIFICATION=C   

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
[1] stringr_1.4.0  purrr_0.3.4    dplyr_1.0.8    xml2_1.3.3     metacore_0.1.0

loaded via a namespace (and not attached):
 [1] Rcpp_1.0.8       rstudioapi_0.13  knitr_1.36       magrittr_2.0.2   tidyselect_1.1.2 R6_2.5.1        
 [7] rlang_1.0.1      fastmap_1.1.0    fansi_1.0.2      blob_1.2.1       tools_4.0.2      xfun_0.31       
[13] utf8_1.2.2       cli_3.3.0        DBI_1.1.0        htmltools_0.5.2  ellipsis_0.3.2   yaml_2.2.1      
[19] digest_0.6.29    readxl_1.3.1     assertthat_0.2.1 tibble_3.1.6     lifecycle_1.0.1  crayon_1.5.0    
[25] tidyr_1.2.0      vctrs_0.3.8      evaluate_0.14    glue_1.6.1       rmarkdown_2.14   stringi_1.7.6   
[31] compiler_4.0.2   pillar_1.7.0     cellranger_1.1.0 generics_0.1.2   pkgconfig_2.0.3 
```

**Comment by mstackhouse** [comment]
I'd like to see #45 updated as well given that we're shooting for a CRAN release. We want to make sure docs are fairly tidied up.

**Comment by mstackhouse** [comment]
> > 
> 
> @statasaurus look at me being a bad reviewer...
> 
> ```
> R version 4.0.2 (2020-06-22)
> Platform: x86_64-pc-linux-gnu (64-bit)
> Running under: Ubuntu 18.04.6 LTS
> 
> Matrix products: default
> BLAS:   /usr/lib/x86_64-linux-gnu/openblas/libblas.so.3
> LAPACK: /usr/lib/x86_64-linux-gnu/libopenblasp-r0.2.20.so
> 
> locale:
>  [1] LC_CTYPE=C.UTF-8       LC_NUMERIC=C           LC_TIME=C.UTF-8        LC_COLLATE=C.UTF-8     LC_MONETARY=C.UTF-8   
>  [6] LC_MESSAGES=C.UTF-8    LC_PAPER=C.UTF-8       LC_NAME=C              LC_ADDRESS=C           LC_TELEPHONE=C        
> [11] LC_MEASUREMENT=C.UTF-8 LC_IDENTIFICATION=C   
> 
> attached base packages:
> [1] stats     graphics  grDevices utils     datasets  methods   base     
> 
> other attached packages:
> [1] stringr_1.4.0  purrr_0.3.4    dplyr_1.0.8    xml2_1.3.3     metacore_0.1.0
> 
> loaded via a namespace (and not attached):
>  [1] Rcpp_1.0.8       rstudioapi_0.13  knitr_1.36       magrittr_2.0.2   tidyselect_1.1.2 R6_2.5.1        
>  [7] rlang_1.0.1      fastmap_1.1.0    fansi_1.0.2      blob_1.2.1       tools_4.0.2      xfun_0.31       
> [13] utf8_1.2.2       cli_3.3.0        DBI_1.1.0        htmltools_0.5.2  ellipsis_0.3.2   yaml_2.2.1      
> [19] digest_0.6.29    readxl_1.3.1     assertthat_0.2.1 tibble_3.1.6     lifecycle_1.0.1  crayon_1.5.0    
> [25] tidyr_1.2.0      vctrs_0.3.8      evaluate_0.14    glue_1.6.1       rmarkdown_2.14   stringi_1.7.6   
> [31] compiler_4.0.2   pillar_1.7.0     cellranger_1.1.0 generics_0.1.2   pkgconfig_2.0.3 
> ```

I didn't read - this is my bad.

**Comment by mstackhouse** [comment]
> I'd like to see #45 updated as well given that we're shooting for a CRAN release. We want to make sure docs are fairly tidied up.

This, gitignore .DS_store, and then I'm good as long as @rossfarrugia can confirm #53  is good

**Comment by mstackhouse** [comment]
@statasaurus Can you run readme.Rmd and push? and think we can get the PNGs updated?

**Comment by statasaurus** [comment]
I can rerun the README, but @MayaGans has all the png's she will need to update them

**Comment by mstackhouse** [comment]
@statasaurus I'm going to approve so we can get the CRAN release out. But we'll come back for the PNGs

**Review by mstackhouse — APPROVED** [review]

**Pull request merged**